api/views.py

```diff
@@ -416,3 +418,30 @@ class Accounts(generics.RetrieveAPIView):
     def get(self, request):
         serializer = AccountSerializer(request.user)
         return response.Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ContactUs(generics.CreateAPIView):
+    serializer_class = ContactUsSerializer
+
+    def post(self, request):
+        email = request.data["email"]
+        name = request.data["name"]
+        message = request.data["message"]
+        serializer = self.serializer_class(
+            context={"email": email, "name": name, "message": message}
+        )
+        return response.Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
I don't think you need to pull those out of the request and pass them in with the context. You should be able to pass the request